### PR TITLE
Guppy sequ kit

### PR DESCRIPTION
**Pull request by DataSpott**
Thought about possibility of adding other barcode-kits, but looking through them no others looked really relevant. Only one I missed in nanozoo/guppy-container was the guppy .cfg-file for the 96-rapid barcoding kit. Couldn´t find information about this file so far either.

### CODE REVIEW

File: poreCov.nf [hunk 1]
```diff
@@ -63,10 +63,10 @@ println "_______________________________________________________________________
 
 // Log infos based on user inputs
 if ( params.help ) { exit 0, helpMSG() }
-    defaultMSG()
-if ( params.primerV.matches('V1200') ) { v1200_MSG() }
-if ( params.fast5 || workflow.profile.contains('test_fast5') ) { basecalling() }
-if ( params.rki ) { rki() }
+//    defaultMSG()
```
Comment (replikation) [inline poreCov.nf:66]: @DataSpott why is this commented out?

File: workflows/process/collect_fastq.nf [hunk 2]
```diff
@@ -5,16 +5,28 @@ process collect_fastq {
     output:
         tuple val(name), path("*.fastq.gz"), emit: reads
     script:
+        if (params.rapid) {
+            guppy_arrangement_files = 'barcode_arrs_rbk4.cfg'
```
Comment (replikation) [inline workflows/process/collect_fastq.nf:9]: @DataSpott this needs the rbk file that would support 96 barcodes (check newest guppy)

File: workflows/process/guppy.nf [hunk 3]
```diff
@@ -36,7 +36,18 @@ process guppy_gpu {
     output:
         tuple val(name), path("*.fastq.gz"), emit: reads
         tuple val(name), path("fastq_tmp/*.txt"), emit: summary
-    script:
+    script:       
+        if (params.rapid) {
+            guppy_arrangement_files = 'barcode_arrs_rbk4.cfg'
```
Comment (replikation) [inline workflows/process/guppy.nf:41]: @DataSpott this needs the rbk file that would support 96 barcodes (check newest guppy)

**Review by replikation — CHANGES_REQUESTED** [review]
* please resolve issues pointed out.

File: poreCov.nf (hunk 1 above)
Comment (DataSpott) [reply poreCov.nf]: Forgot to deletes this after moving to new section

**Review by replikation — APPROVED** [review]

**Comment by replikation** [comment]
* currently cuda issues on guppy.

**Comment by DataSpott** [comment]
Implemented support for Rapid-barcoding-kit. 
Update to guppy-gpu (and therefore also guppy-cpu) -container rolled back, due to issues between guppy v.4.5.4 and CUDA with following error-message in .command.log:
 
    ONT Guppy basecalling software version 4.5.4+66c1a7753
    config file:        /opt/ont/guppy/data/dna_r9.4.1_450bps_hac.cfg
    model file:         /opt/ont/guppy/data/template_r9.4.1_450bps_hac.jsn
    input path:         20210423_covid_rapBar_test2
    save path:          fastq_tmp
    chunk size:         2000
    chunks per runner:  512
    minimum qscore:     9
    records per file:   4000
    num basecallers:    4
    gpu device:         auto
    kernel path:        
    runners per device: 4

    [guppy/error] main: CUDA error at /builds/ofan/ont_core_cpp/ont_core/common/cuda_common.cpp:203: CUDA_ERROR_COMPAT_NOT_SUPPORTED_ON_DEVICE
    [guppy/warning] main: An error has occurred. Aborting.

**Review by replikation — APPROVED** [review]
* thx

**Pull request merged**